src/connectedk8s/azext_connectedk8s/_utils.py

```diff
@@ -98,12 +104,14 @@ def pull_helm_chart(registry_path, kube_config, kube_context, helm_client_locati
     response_helm_chart_pull = subprocess.Popen(cmd_helm_chart_pull, stdout=PIPE, stderr=PIPE)
     _, error_helm_chart_pull = response_helm_chart_pull.communicate()
     if response_helm_chart_pull.returncode != 0:
+        if chart_path_name == 'AzureArcCharts':
+            chart_path_name = 'azure-arc' 
```

Comment:
check the naming here everywhere , use "chart_path_name" is relevant only for get_chart_path() method. For others it should say something like chart_name